app/content/blog/wiki/price-sniper-and-item-price-alerts.md

```
@@ -0,0 +1,70 @@
+- Or pick `below` to be alerted when the marketboard price is below your price region wide.
+- If you want to only be alerted on HQ versions or all version of the item.
+- Choose the `Quantity` button if you want to be alerted on quantity instead of price.
+
+<img width="710" alt="image" src="https://user-images.githubusercontent.com/17516896/210079936-5a6a6472-b2d6-400a-89c4-2f87258e9a6e.png">
+<img width="606" alt="image" src="https://github.com/ff14-advanced-market-search/saddlebag-with-pockets/assets/17516896/dbd80afc-1687-4cf0-a27c-8ef2203d6adf">
+
+3. Click add to add this to your json data.
+
+4. When you are done copy your json data to your clipboard. It may look like the following:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)